src/test/fedora-31/Dockerfile.in

```diff
@@ -24,6 +24,5 @@ COPY ceph.spec.in /root/
 # build dependencies
 RUN dnf install -y which ; cd /root ; ./install-deps.sh
 # development tools
-# nc is required to run make check on firefly only (giant+ do not use nc)
-RUN dnf install -y ccache valgrind gdb git python-virtualenv gdisk kpartx jq sudo xmlstarlet parted nc
+RUN dnf install -y gdb git sudo
```

Comment:
install-deps.sh tests for the presence of git and errors if it can't find it and also calls sudo so I think we need to look at the order here?

Reply:
thanks Brad! i reorder them.

could you please take another look?

Reply:
> thanks Brad! i reorder them.
> 
> could you please take another look?

Definitely.

Reply:
looks good mate